config/webhook/admission_webhook.yaml

```diff
@@ -79,7 +79,16 @@ spec:
           mountPath: /etc/certs
           readOnly: true
         securityContext:
+          allowPrivilegeEscalation: false
           readOnlyRootFilesystem: true
+          runAsNonRoot: true
+          runAsUser: 2000
+          runAsGroup: 2000
```

Comment:
https://github.com/kubernetes-sigs/gateway-api/blob/6fee7b5cd91c848995b359dfb5c84de4825c6d05/docker/Dockerfile.webhook#L30

`65532`

Reply:
Interesting, I just copied the userid from certificate config.  Should they both be updated to `65532`?

Reply:
This would be unique to our admission-server image.